src/main/java/com/marginallyclever/makelangelo/turtle/turtleRenderer/TurtleRenderFactory.java

```diff
@@ -11,18 +12,27 @@
  * @author Dan Royer
  */
 public enum TurtleRenderFactory {
-	DEFAULT("Default", new DefaultTurtleRenderer()),
-	BARBER_POLE("Barber pole", new BarberPoleTurtleRenderer()),
-	SEPARATE_LOOP("Separate loops",new SeparateLoopTurtleRenderer()),
-	MARLIN_SIM("Marlin simulation",new MarlinSimulationVisualizer());
+	DEFAULT("Default", new DefaultTurtleRenderer(),Translator.get("TurtleRenderFactory.DEFAULT")),// not coder friendly but allow later the CI test not to miss this translation keys.
+	BARBER_POLE("Barber pole", new BarberPoleTurtleRenderer(),Translator.get("TurtleRenderFactory.BARBER_POLE")),
+	SEPARATE_LOOP("Separate loops",new SeparateLoopTurtleRenderer(),Translator.get("TurtleRenderFactory.SEPARATE_LOOP")),
+	MARLIN_SIM("Marlin simulation",new MarlinSimulationVisualizer(),Translator.get("TurtleRenderFactory.MARLIN_SIM"));
```

Comment:
When does this run?  I mean, how do we know it runs after Translator has loaded a language?

Reply:
how do we *guarantee*

Reply:
no it won't, you are right the correct language may not be loaded before the enum is loaded :'(

Reply:
So my apologies for this proposal which does not guarantee the correct translation.

As I'm out of ideas, and in order to keep things simple in the code, I'm not going to use a Translator.get("...") expression recognized by the IC test for missing translation keys.

Therefore, for the translation of the items of this enumeration in the menu, if translation keys are missing, will not be detected by the CI test.

Reply:
Enum sounds good to make sure the classes exist at compile time.  A factory and a `testMakeOneOfEach()` would do the same job, plus the factory will definitely run after initialization and use less ram/startup time if never called at all.

Reply:
What you can do is simply put the Key in the enum and when the translation in needed, call the translator with the key, like : 
```
DEFAULT("Default", new DefaultTurtleRenderer(), "TurtleRenderFactory.DEFAULT")
```

If the CI test is a worry, don't worry :) The regexp will on catch `Translator.get("XXX")`, a pattern with double quote, so it won't hurt.

Reply:
Thanks, much more logical indeed to keep the translation keys in the enum